forms/jqm/element.js

```diff
@@ -158,6 +166,10 @@ define(function (require) {
 
     },
 
+    onChangeClass: function () {
+      this.$el.attr('class', this.model.attributes['class']);
```

Comment:
Yeah, that fell through. If this ever becomes a problem, we'll need to get all kinds of weird to solve this (e.g. MutationObservers).